hydrolib/core/io/mdu/models.py

```diff
@@ -176,8 +176,10 @@ def is_intermediate_link(self) -> bool:
 class Lateral(INIBasedModel):
     _header: Literal["Lateral"] = "Lateral"
     id: str
-    name: str
-    nodeId: str
+    name: str = ""
+    nodeId: Optional[str]
+    branchId: Optional[str]
+    chainage: Optional[float]
```

Comment:
I believe a Lateral should have either nodeId OR branchId + chainage. Is it possible to validate this?